common/utils/src/main/java/org/graalvm/buildtools/utils/NativeImageUtils.java

```diff
@@ -122,7 +122,10 @@ public static void checkVersion(String requiredVersion, String versionToCheck) {
         if (versionToCheck.contains("GraalVM Runtime Environment")) {
             return; // later than 22.3.1 (e.g., GraalVM for JDK 17 / GraalVM for JDK 20)
         }
-        if (versionToCheck.startsWith("GraalVM dev") || versionToCheck.startsWith("native-image dev")) {
+        if (!versionToCheck.startsWith("GraalVM") && !versionToCheck.startsWith("native-image") && !versionToCheck.startsWith("Oracle GraalVM")) {
```

Comment:
Let's start with the latest format first (CE and Oracle GraalVM print `native-image ...`  now:

```suggestion
        if (!versionToCheck.startsWith("native-image") && !versionToCheck.startsWith("GraalVM") && !versionToCheck.startsWith("Oracle GraalVM")) {
```